src/Tabs/docs/examples.js

```diff
@@ -0,0 +1,85 @@
+const items = [1, 2, 3, 4, 5].map(index => ({
+  id: index,
+  title: `item ${index}`,
+}));
+
+const sideContent = `
+<div>
+  <div>This is a</div>
+  <div>side content</div>
+</div>
+`;
+
+export const base = `
+class TabsExample extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      activeId: 1,
+    };
+    this.change = this.change.bind(this);
+  }
+
+  change(value) { this.setState({ activeId: value.id }) }
+
+  render() {
+    const { activeId } = this.state;
+    return (
+      <Tabs
+        dataHook="story-tabs"
+        activeId={activeId}
+        onClick={this.change}
+        items={${JSON.stringify(items)}}
+      />
+    );
+  }
+}
+`;
+
+export const hasDivider = `
+  <Tabs
+    dataHook="story-tabs"
+    items={${JSON.stringify(items)}}
+    activeId={2}
+    hasDivider={false}
+  />
+`;
+
+export const types = `
+  <div>
+    <h3>Default</h3>
```

Comment:
I would suggest to the consumers to use `<Heading/>`:
```suggestion
    <Heading appearance="H3">Default</Heading>
```